### PR TITLE
Fixed Missing default Image & Slide-arrows in product card store/order

### PR DESCRIPTION
**Pull request by Sourav635**
## Description
Assigned a default image for product with no image.
Fixed Visibility of Slide-Arrows

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Changes and Fixes
- A default product image will appear whenever image is not provided for a certain product.
- Slide arrows for switching between multiple images of a product will not appear if product contain single image.

## Screenshots 
image
image

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Looks good on large screens
- [x] Looks good on mobiles

Affected Apps
- [x] Store
    - [x] Order 
    - [ ] SEO
    - [ ] Menu
    - [ ] Checkout
    - [ ] Add to Cart logic
    - [ ] Location Logic
    - [ ] Product page
    - [ ] Login
    - [ ] Payment
    - [ ] Kiosk
- [ ] Admin
    - [ ] Order
    - [ ] CRM
    - [ ] Product app
    - [ ] Analytics
    - [ ] Settings
    - [ ] Brand
    - [ ] Config Builder
    - [ ] Coupons
    - [ ] Locations
    - [ ] Users
    - [ ] Inventory
- [ ] Server
    - [ ] Payment
    - [ ] Email
    - [ ] SMS
    - [ ] Notifications
    - [ ] Integration
    - [ ] Hasura auth
- [ ] Template Engine

### CODE REVIEW

File: store/src/sections/order/index.js [hunk 1]
```diff
@@ -288,6 +288,19 @@ const ProductWithIntersection = ({
...
+   //    eachProduct.assets.images[0] == undefined ||
+   //       eachProduct.assets.images[0] === null
+   // )
+   if (
+      eachProduct.assets.images[0] === undefined ||
+      eachProduct.assets.images[0] === null
+   ) {
+      eachProduct.assets.images[0] =
+         'https://d3i0trfokco6a2.cloudfront.net/TPO-396.png'
+   }
```
Comment (FAsami) [inline store/src/sections/order/index.js:303]: - The source images should not be changed and the default image should be config based. 
- Remove unnecessary comments that you think will not be required in the future.

Comment (shubhamz950) [reply store/src/sections/order/index.js:303]: @Sourav635 @FAsami is it resolved ?

**Review by shubhamz950 — APPROVED** [review]

File: store/src/components/product_card.jsx [hunk 2]
```diff
@@ -136,9 +138,47 @@ export const ProductCard = props => {
...
                         infinite={autoPlaySlider}
                         autoplay={autoPlaySlider}
+                        {...properties}
                      >
                         {data.assets.images.map((each, index) => {
-                           return (
+
+                           // each = noProductImage
+                           // console.log("abcde",noProductImage)
+                           console.log("EACH",each)
```
Comment (FAsami) [inline store/src/components/product_card.jsx:147]: Remove unused code

File: store/src/components/product_card.jsx [hunk 3]
```diff
@@ -136,9 +138,47 @@ export const ProductCard = props => {
...
+                           if (each === undefined || each === null)
```
Comment (FAsami) [inline store/src/components/product_card.jsx:148]: Could be checked before mapping images with `data.assets.images.length`

**Pull request merged**